streams.yml

```diff
@@ -45,6 +45,17 @@ rhel-9-golang:
   upstream_image_mirror:
   - registry.ci.openshift.org/ocp-private/builder-priv:rhel-9-golang-1.21-openshift-{MAJOR}.{MINOR}
 
+rhel-9-golang-1.22:
```

Comment:
For 4.17, think this can be `rhel-9-golang` immediately.

Reply:
I would like to go as is right now, since rhel8 builder isn't available. That way CI would have it available, and when rhel8 build is available, i will build the builder and update default streams